src/Symfony/Component/Notifier/Bridge/Smsbox/.gitattributes

```diff
@@ -0,0 +1,4 @@
+/Tests export-ignore
+/phpunit.xml.dist export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
```

Comment:
please add a newline to make the red pictogram disapear

Reply:
fixed